README.md

```diff
@@ -37,18 +38,18 @@ class Book {
 	}
 }
 
-let trainDreams = new Book(
-	'Train Dreams',
-	'Denis Johnson',
-	'Historical Fiction',
-	'2002'
+let lordOfTheRings = new Book(
+	'The Lord of the Rings',
+	'J.R.R. Tolkien',
+	'Fantasy',
+	'1954'
 );
 ```
 
-Some of these properties might be shared between many books, though. Authors can
-write many books. Many books fall into the same genres. The Object Oriented
+Some of these properties might be shared between many books, though. Tolkien has
+written other books. Many books fall into the same genre. The Object Oriented
```

Comment:
"many of her books fall..." (assuming we're still talking about Tolkien, if not, then clarify)